.travis.yml

```diff
@@ -8,7 +8,10 @@ gemfile:
   - gemfiles/rails_5.1.gemfile
   - gemfiles/rails_5.2.gemfile
   - gemfiles/rails_6.0.gemfile
+  - gemfiles/rails_6.1.gemfile
 matrix:
   exclude:
     - rvm: 2.4.4
       gemfile: gemfiles/rails_6.0.gemfile
+    - rvm: 2.4.4
+      gemfile: gemfiles/rails_6.1.gemfile
```

Comment:
woohoo, thanks for adding this!

Reply:
no problem! wondering if i did something wrong in the setup though. the tests for rails 6.1 are running in full and passing on my machine, but the test suite seems to be running only some tests and exiting with code 1

Reply:
hm, yeah, i see that travis is showing something weird. 

since it's only showing that it ran half the tests, that tells me that there is an error that we're not seeing. the only thing i can think of is that it's thrown into `stdout` and our tests are hiding it. i think the hiding is done by something like this https://github.com/ilyakatz/data-migrate/blob/master/spec/data_migrate/migration_context_spec.rb#L75

2) I ran these tests on your branch and i see a couple of failures that seem relevant. do you think that they are legit?

```
Failures:

  1) DataMigrate::DatabaseTasks migrations load_schema_current loads the current schema file
     Failure/Error: subject.load_schema_current
     
       #<ActiveRecord::DatabaseConfigurations:0x00005565be6df6e0 @configurations=[]> received :configs_for with unexpected arguments
         expected: ({:env_name=>"test"})
              got: ({:env_name=>"development"})
       Diff:
       @@ -1 +1 @@
       -[{:env_name=>"test"}]
       +[{:env_name=>"development"}]
       
        Please stub a default value first if message might be received with other args as well. 
     # ./spec/data_migrate/database_tasks_spec.rb:105:in `block (4 levels) in <top (required)>'

  2) DataMigrate::DatabaseTasks migrations load_schema_current loads schema file that has not been update with latest data migrations
     Failure/Error: subject.load_schema_current
     
       #<ActiveRecord::DatabaseConfigurations:0x00005565be6df6e0 @configurations=[]> received :configs_for with unexpected arguments
         expected: ({:env_name=>"test"})
              got: ({:env_name=>"development"})
       Diff:
       @@ -1 +1 @@
       -[{:env_name=>"test"}]
       +[{:env_name=>"development"}]
       
        Please stub a default value first if message might be received with other args as well. 
     # ./spec/data_migrate/database_tasks_spec.rb:117:in `block (4 levels) in <top (required)>'
```

Reply:
i get those locally too, but the issue is the environment name (locally it's `development`, in test it is `test`). if you run the rails-6.0 specs locally, similar issues come up.

besides, if this were the issue, i would expect to see it in the travis output.

Reply:
i'll take a look at the migration_context tests and see if i can duplicate the stdout catching